Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -2531,7 +2529,6 @@ auto B3IRGenerator::addArraySet(uint32_t typeIndex, ExpressionType arrayref, Exp
     }
 
     // FIXME: Emit this inline.
-    // https://bugs.webkit.org/show_bug.cgi?id=245405
```

Comment:
why is this removed?

Reply:
looks like a mistaken deletion, I put it back